docker/Dockerfile

```diff
@@ -15,7 +15,7 @@ FROM ethereum/solc:0.6.4-alpine as bsc-genesis
 RUN apk add --d --no-cache ca-certificates npm nodejs bash alpine-sdk
 
 RUN git clone https://github.com/binance-chain/bsc-genesis-contract.git /root/genesis \
-    && rm /root/genesis/package-lock.json && cd /root/genesis && npm install
+    && cd /root/genesis && npm install
```

Comment:
what is the motivation of this?

Reply:
this was from rebase https://github.com/bnb-chain/bsc/pull/1361/commits/425843966f85fe9c10ecdd0f3135bbd5710ec9d9